src/systems/userinput/userinput.js

```diff
@@ -163,7 +163,8 @@ AFRAME.registerSystem("userinput", {
   },
 
   toggleSet(set, value) {
-    this.pendingSetChanges.push({ set, value });
+    const b = !!value;
```

Comment:
(We only care about whether the `value` passed in is truthy.)

Reply:
Not awful the sanitize this input, but I would ditch the temp variable